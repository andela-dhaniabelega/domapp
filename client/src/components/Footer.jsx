import React from 'react';

const Footer = () => ({
	render() {
		return (
			<div>
				<footer className="page-footer">
					<div className="container">
						<div className="row">
							<div className="col l6 s12">
								<h5 className="white-text">Domapp</h5>
								<p className="grey-text text-lighten-4 footer-text">
									Premium Document Management System
									Domapp provides the most efficient Document Management
									Algorithms on the market. Manage your documents both
									locally and on the Cloud.
								</p>
							</div>
							<div className="col l4 offset-l2 s12">
								<h5 className="white-text">Quick Links</h5>
								<ul className="footer-text">
									<li>
										<a className="grey-text text-lighten-3" href="/mydocuments">My Documents</a>
									</li>
									<li>
										<a className="grey-text text-lighten-3" href="/settings">Settings</a>
									</li>
									<li>
										<a className="grey-text text-lighten-3" href="/documents">All Documents</a>
									</li>
									<li>
										<a className="grey-text text-lighten-3" href="/logout">Log Out</a>
									</li>
								</ul>
							</div>
						</div>
					</div>
					<div className="footer-copyright">
						<div className="container">
            Â© 2017 Copyright Domapp
							<a
								className="grey-text text-lighten-4 right all-rights"
								href="#!"
							>
								All Rights Reserved
							</a>
						</div>
					</div>
				</footer>
			</div>
		);
	}
});
export default Footer;
