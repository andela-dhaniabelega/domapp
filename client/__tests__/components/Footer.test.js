import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../src/components/Footer';

describe('Footer', () => {
	it('renders a snapshot', () => {
		const wrapper = shallow(<Footer />);
		expect(wrapper).toMatchSnapshot();
	});
});
