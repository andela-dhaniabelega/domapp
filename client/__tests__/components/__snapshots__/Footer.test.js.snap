// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders a snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <footer
        className="page-footer"
    >
        <div
            className="container"
        >
            <div
                className="row"
            >
                <div
                    className="col l6 s12"
                >
                    <h5
                        className="white-text"
                    >
                        Domapp
                    </h5>
                    <p
                        className="grey-text text-lighten-4 footer-text"
                    >
                        Premium Document Management System Domapp provides the most efficient Document Management Algorithms on the market. Manage your documents both locally and on the Cloud.
                    </p>
                </div>
                <div
                    className="col l4 offset-l2 s12"
                >
                    <h5
                        className="white-text"
                    >
                        Quick Links
                    </h5>
                    <ul
                        className="footer-text"
                    >
                        <li>
                            <a
                                className="grey-text text-lighten-3"
                                href="/mydocuments"
                            >
                                My Documents
                            </a>
                        </li>
                        <li>
                            <a
                                className="grey-text text-lighten-3"
                                href="/settings"
                            >
                                Settings
                            </a>
                        </li>
                        <li>
                            <a
                                className="grey-text text-lighten-3"
                                href="/documents"
                            >
                                All Documents
                            </a>
                        </li>
                        <li>
                            <a
                                className="grey-text text-lighten-3"
                                href="/logout"
                            >
                                Log Out
                            </a>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        <div
            className="footer-copyright"
        >
            <div
                className="container"
            >
                © 2017 Copyright Domapp
                <a
                    className="grey-text text-lighten-4 right all-rights"
                    href="#!"
                >
                    All Rights Reserved
                </a>
            </div>
        </div>
    </footer>
</div>,
  "nodes": Array [
    <div>
      <footer
            className="page-footer"
      >
            <div
                  className="container"
            >
                  <div
                        className="row"
                  >
                        <div
                              className="col l6 s12"
                        >
                              <h5
                                    className="white-text"
                              >
                                    Domapp
                              </h5>
                              <p
                                    className="grey-text text-lighten-4 footer-text"
                              >
                                    Premium Document Management System Domapp provides the most efficient Document Management Algorithms on the market. Manage your documents both locally and on the Cloud.
                              </p>
                        </div>
                        <div
                              className="col l4 offset-l2 s12"
                        >
                              <h5
                                    className="white-text"
                              >
                                    Quick Links
                              </h5>
                              <ul
                                    className="footer-text"
                              >
                                    <li>
                                          <a
                                                className="grey-text text-lighten-3"
                                                href="/mydocuments"
                                          >
                                                My Documents
                                          </a>
                                    </li>
                                    <li>
                                          <a
                                                className="grey-text text-lighten-3"
                                                href="/settings"
                                          >
                                                Settings
                                          </a>
                                    </li>
                                    <li>
                                          <a
                                                className="grey-text text-lighten-3"
                                                href="/documents"
                                          >
                                                All Documents
                                          </a>
                                    </li>
                                    <li>
                                          <a
                                                className="grey-text text-lighten-3"
                                                href="/logout"
                                          >
                                                Log Out
                                          </a>
                                    </li>
                              </ul>
                        </div>
                  </div>
            </div>
            <div
                  className="footer-copyright"
            >
                  <div
                        className="container"
                  >
                        © 2017 Copyright Domapp
                        <a
                              className="grey-text text-lighten-4 right all-rights"
                              href="#!"
                        >
                              All Rights Reserved
                        </a>
                  </div>
            </div>
      </footer>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Footer />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Object {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "render": [Function],
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <footer
                    className="page-footer"
          >
                    <div
                              className="container"
                    >
                              <div
                                        className="row"
                              >
                                        <div
                                                  className="col l6 s12"
                                        >
                                                  <h5
                                                            className="white-text"
                                                  >
                                                            Domapp
                                                  </h5>
                                                  <p
                                                            className="grey-text text-lighten-4 footer-text"
                                                  >
                                                            Premium Document Management System Domapp provides the most efficient Document Management Algorithms on the market. Manage your documents both locally and on the Cloud.
                                                  </p>
                                        </div>
                                        <div
                                                  className="col l4 offset-l2 s12"
                                        >
                                                  <h5
                                                            className="white-text"
                                                  >
                                                            Quick Links
                                                  </h5>
                                                  <ul
                                                            className="footer-text"
                                                  >
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/mydocuments"
                                                                      >
                                                                                My Documents
                                                                      </a>
                                                            </li>
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/settings"
                                                                      >
                                                                                Settings
                                                                      </a>
                                                            </li>
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/documents"
                                                                      >
                                                                                All Documents
                                                                      </a>
                                                            </li>
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/logout"
                                                                      >
                                                                                Log Out
                                                                      </a>
                                                            </li>
                                                  </ul>
                                        </div>
                              </div>
                    </div>
                    <div
                              className="footer-copyright"
                    >
                              <div
                                        className="container"
                              >
                                        © 2017 Copyright Domapp
                                        <a
                                                  className="grey-text text-lighten-4 right all-rights"
                                                  href="#!"
                                        >
                                                  All Rights Reserved
                                        </a>
                              </div>
                    </div>
          </footer>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <footer
                    className="page-footer"
          >
                    <div
                              className="container"
                    >
                              <div
                                        className="row"
                              >
                                        <div
                                                  className="col l6 s12"
                                        >
                                                  <h5
                                                            className="white-text"
                                                  >
                                                            Domapp
                                                  </h5>
                                                  <p
                                                            className="grey-text text-lighten-4 footer-text"
                                                  >
                                                            Premium Document Management System Domapp provides the most efficient Document Management Algorithms on the market. Manage your documents both locally and on the Cloud.
                                                  </p>
                                        </div>
                                        <div
                                                  className="col l4 offset-l2 s12"
                                        >
                                                  <h5
                                                            className="white-text"
                                                  >
                                                            Quick Links
                                                  </h5>
                                                  <ul
                                                            className="footer-text"
                                                  >
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/mydocuments"
                                                                      >
                                                                                My Documents
                                                                      </a>
                                                            </li>
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/settings"
                                                                      >
                                                                                Settings
                                                                      </a>
                                                            </li>
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/documents"
                                                                      >
                                                                                All Documents
                                                                      </a>
                                                            </li>
                                                            <li>
                                                                      <a
                                                                                className="grey-text text-lighten-3"
                                                                                href="/logout"
                                                                      >
                                                                                Log Out
                                                                      </a>
                                                            </li>
                                                  </ul>
                                        </div>
                              </div>
                    </div>
                    <div
                              className="footer-copyright"
                    >
                              <div
                                        className="container"
                              >
                                        © 2017 Copyright Domapp
                                        <a
                                                  className="grey-text text-lighten-4 right all-rights"
                                                  href="#!"
                                        >
                                                  All Rights Reserved
                                        </a>
                              </div>
                    </div>
          </footer>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Footer />,
}
`;
